services/iam/client.go

```diff
@@ -39,12 +39,18 @@ type IAMClient interface {
 	RemoveAuthorizationResource(resource common.Origin) error
 	RemoveAuthorizationResourceWithContext(ctx context.Context, resource common.Origin) error
 
+	GetAuthorizationResourcesByType(typ string) (resources []common.Origin, err error)
```

Comment:
`typ` swedish version of type?
Is type a reserved word? What about resourceType instead of type?

Reply:
`type` is reserved in go, so that's the reason.
`resourceType` makes sense, I'll change to that.